From bb5b5203bc33e43ff2787cdc4270248c3196d1d0 Mon Sep 17 00:00:00 2001
From: Leo Chang <leochang@codeaurora.org>
Date: Thu, 26 Sep 2013 20:15:47 -0700
Subject: [PATCH] wlan: Quarky functional fail fix

If NL client try to registration,
Kernel driver broadcast KD ready notification to reset
application status.

Change-Id: I31e21730be518f1b6ac6ab7a3aaf7cd6e2d6f3bc
CRs-fixed: 539232
---
 CORE/SVC/src/ptt/wlan_ptt_sock_svc.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/CORE/SVC/src/ptt/wlan_ptt_sock_svc.c b/CORE/SVC/src/ptt/wlan_ptt_sock_svc.c
index ee5b30a..d547ac6 100644
--- a/CORE/SVC/src/ptt/wlan_ptt_sock_svc.c
+++ b/CORE/SVC/src/ptt/wlan_ptt_sock_svc.c
@@ -148,6 +148,11 @@ static void ptt_sock_proc_reg_req(tAniHdr *wmsg, int radio)
    //send reg response message to the application
    rspmsg.ret = ANI_NL_MSG_OK;
    rspmsg.regReq.type = reg_req->type;
+#ifdef WLAN_KD_READY_NOTIFIER
+   /* NL client try to registration
+    * to make sure connection, broadcast READY notification */
+   nl_srv_nl_ready_indication();
+#endif /* WLAN_KD_READY_NOTIFIER */
    /*Save the pid*/    
    pAdapterHandle->ptt_pid = reg_req->pid;   
    rspmsg.regReq.pid= reg_req->pid;
-- 
1.8.2.1

